docs/pages/setup/reference/cli.mdx

```diff
@@ -685,6 +685,54 @@ registered devices using [`tsh mfa ls`](cli.mdx#tsh-mfa-ls).
 $ tsh mfa rm <device-name>
 ```
 
+### tsh exec
+
+Run provided command, by default using a shell, with Teleport-specific environment
+variables set. Can be used to create combined commands.
```

Comment:
I think this should expand on "combined commands." What are these commands combined with?

Reply:
The motivating example is in RFD:

```
aliases:
  "connect": "exec -- -c '$TSH login $0 && $TSH ssh $1'"
```

Here we use `exec` to combine `login` and `ssh`.